tests/lapack/full_matrix_38.output

```diff
@@ -0,0 +1,37 @@
+
+DEAL::Checking matrix 
+4.000e+00  -1.000e+00 
+-1.000e+00 4.000e+00  
+DEAL::Eigenvalues: (5.00000,0.00000) (3.00000,0.00000) 
+DEAL::Right eigenvectors
+(0.71,0.0)(0.71,0.0)
+(-0.71,0.0)(0.71,0.0)
+DEAL::Left eigenvectors
+(0.71,0.0)(0.71,0.0)
+(-0.71,0.0)(0.71,0.0)
+DEAL::
+DEAL::Checking matrix 
+1.000e+00  -1.000e+00 
+1.000e+00  1.000e+00  
+DEAL::Eigenvalues: (1.00000,1.00000) (1.00000,-1.00000) 
+DEAL::Right eigenvectors
+(0.71,0.0)(0.71,0.0)
+(0.0,-0.71)(0.0,0.71)
+DEAL::Left eigenvectors
+(-0.71,0.0)(-0.71,0.0)
+(0.0,0.71)(0.0,-0.71)
+DEAL::
+DEAL::Checking matrix 
+1.000e+00             2.000e+00  
+           1.000e+00             
+                      1.000e+00  
+DEAL::Eigenvalues: (1.00000,0.00000) (1.00000,0.00000) (1.00000,0.00000) 
+DEAL::Right eigenvectors
+(1.0,0.0)(0.0,0.0)(-1.0,0.0)
+(0.0,0.0)(1.0,0.0)(0.0,0.0)
+(0.0,0.0)(0.0,0.0)(1.1e-16,0.0)
```

Comment:
Huh - that's what LAPACK does for defective matrices.

Reply:
It seems so. That aside, I'm curious how this works for `numdiff`. I might need to print complex numbers manually if the braces `(` and `)` prevent the detection of small numeric values.

Reply:
This should be fine. We tell `numdiff` that those characters are digit separators:

https://github.com/dealii/dealii/blob/master/cmake/scripts/run_test.sh#L148